2025-04-02 13:50:22,510 INFO    MainThread:96971 [wandb_setup.py:_flush():67] Current SDK version is 0.19.9
2025-04-02 13:50:22,510 INFO    MainThread:96971 [wandb_setup.py:_flush():67] Configure stats pid to 96971
2025-04-02 13:50:22,510 INFO    MainThread:96971 [wandb_setup.py:_flush():67] Loading settings from /Users/noaschwartz_wandb/.config/wandb/settings
2025-04-02 13:50:22,510 INFO    MainThread:96971 [wandb_setup.py:_flush():67] Loading settings from /Users/noaschwartz_wandb/multi-node/wandb/settings
2025-04-02 13:50:22,510 INFO    MainThread:96971 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-04-02 13:50:22,510 INFO    MainThread:96971 [wandb_init.py:setup_run_log_directory():662] Logging user logs to /Users/noaschwartz_wandb/multi-node/wandb/run-20250402_135022-1drfs6cx/logs/debug.log
2025-04-02 13:50:22,510 INFO    MainThread:96971 [wandb_init.py:setup_run_log_directory():663] Logging internal logs to /Users/noaschwartz_wandb/multi-node/wandb/run-20250402_135022-1drfs6cx/logs/debug-internal.log
2025-04-02 13:50:22,510 INFO    MainThread:96971 [wandb_init.py:init():781] calling init triggers
2025-04-02 13:50:22,510 INFO    MainThread:96971 [wandb_init.py:init():786] wandb.init called with sweep_config: {}
config: {'_wandb': {'code_path': 'code/ai_train.py'}}
2025-04-02 13:50:22,510 INFO    MainThread:96971 [wandb_init.py:init():809] starting backend
2025-04-02 13:50:22,510 INFO    MainThread:96971 [wandb_init.py:init():813] sending inform_init request
2025-04-02 13:50:22,553 INFO    MainThread:96971 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-04-02 13:50:22,553 INFO    MainThread:96971 [wandb_init.py:init():823] backend started and connected
2025-04-02 13:50:22,556 INFO    MainThread:96971 [wandb_init.py:init():915] updated telemetry
2025-04-02 13:50:22,556 INFO    MainThread:96971 [wandb_init.py:init():939] communicating run to backend with 60.0 second timeout
2025-04-02 13:50:53,038 WARNING MainThread:96971 [wandb_init.py:init():1534] [no run ID] interrupted
Traceback (most recent call last):
  File "/Users/noaschwartz_wandb/.pyenv/versions/3.9.7/envs/weave/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1530, in init
    return wi.init(run_settings, run_config)
  File "/Users/noaschwartz_wandb/.pyenv/versions/3.9.7/envs/weave/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 965, in init
    result = wait_with_progress(
  File "/Users/noaschwartz_wandb/.pyenv/versions/3.9.7/envs/weave/lib/python3.9/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 24, in wait_with_progress
    return wait_all_with_progress(
  File "/Users/noaschwartz_wandb/.pyenv/versions/3.9.7/envs/weave/lib/python3.9/site-packages/wandb/sdk/mailbox/wait_with_progress.py", line 87, in wait_all_with_progress
    return asyncio_compat.run(progress_loop_with_timeout)
  File "/Users/noaschwartz_wandb/.pyenv/versions/3.9.7/envs/weave/lib/python3.9/site-packages/wandb/sdk/lib/asyncio_compat.py", line 30, in run
    return future.result()
  File "/Users/noaschwartz_wandb/.pyenv/versions/3.9.7/lib/python3.9/concurrent/futures/_base.py", line 440, in result
    self._condition.wait(timeout)
  File "/Users/noaschwartz_wandb/.pyenv/versions/3.9.7/lib/python3.9/threading.py", line 312, in wait
    waiter.acquire()
KeyboardInterrupt
2025-04-02 13:50:53,635 INFO    MsgRouterThr:96971 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 1 handles.
